<launch>
  <arg name="node_name" default="adding_images" />

  <arg name="image1" default="image1" doc="The image topic. Should be remapped to the name of the real image topic." />
  <arg name="image2" default="image1" doc="The image topic. Should be remapped to the name of the real image topic." />

  <arg name="use_camera_info" default="false" doc="Indicates that the camera_info topic should be subscribed to to get the default input_frame_id. Otherwise the frame from the image message will be used." />
  <arg name="debug_view" default="true" doc="Specify whether the node displays a window to show edge image" />
  <arg name="alpha" value="0.5" doc="Weight of the first array elements" />
  <arg name="gamma" value="0" doc="Scalar added to each sum" />

  <!-- adding_images.cpp -->
  <node name="$(arg node_name)" pkg="opencv_apps" type="adding_images" >
    <remap from="image" to="$(arg image)" />
    <param name="use_camera_info" value="$(arg use_camera_info)" />
    <param name="debug_view" value="$(arg debug_view)" />
    <param name="alpha" value="$(arg alpha)" />
    <param name="gamma" value="$(arg gamma)" />
    <param name="approximate_sync" value="true" />
  </node>
</launch>
